packages/story-editor/src/components/floatingMenu/elements/borderRadius.js

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { memo } from '@googleforcreators/react';
+import { Icons } from '@googleforcreators/design-system';
+import { __ } from '@googleforcreators/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { IconButton, Separator } from './shared';
+
+const BorderRadius = memo(function BorderRadius() {
+  return (
+    <>
+      <IconButton
+        Icon={Icons.Corner}
+        title={__('Change border radius', 'web-stories')}
+        onClick={() => {}}
+      />
+      <Separator />
```

Comment:
This element includes its own separator, because this element and separator is optional, depending on whether the current shape can have border-radius at all (if non-rectangular, this button and separator will be hidden).